browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2395,7 +2395,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 				controls['label'] = label;
 			}
-
+			var disabled;
```

Comment:
initial value should be read from JSON:
see below we do:
`if (data.enabled === 'false' || data.enabled === false)`
we should
`var disabled = data.enabled === false;`  (it it no longer a string). Ant then use disabled variable below

Reply:
Yes good catch, i will update it